app/controllers/DvrController.scala

```diff
@@ -121,7 +121,7 @@ class DvrController @Inject()(
           dvrCaseManagement
             .getDvrDocument(link.uarn, valuationId, link.submissionId, fileRef)
             .map { document =>
-              Result(ResponseHeader(200, document.headers.updated(CONTENT_DISPOSITION, s"""attachment;filename="Untitled Document"""")),
+              Result(ResponseHeader(200, document.headers.updated(CONTENT_DISPOSITION, s"""attachment;filename="Untitled Document.pdf"""")),
```

Comment:
is there really no better, meaningful file name? perhaps use `${link.submissionId}.pdf`